### PR TITLE
SPARQL-star Update

### PR DESCRIPTION
**Pull request by hartig**
This PR is meant to address #14. That is, It adds the necessary content for Section 5 (SPARQL-star Update).

Notice that this PR provides not only the relevant parts that define SPARQL-star Update but also an informal overview with several examples. Currently, this overview is in the first subsection of Section 5. However, this overview may also be moved out of Section 5 into the general overview/primer part of the report. In other words, this text may actually be used to address #94.

@afs can you please take a look at this PR.

/cc @pchampin

(the changes in this PR are copied over from #97 which was based on the old organization of files in the repo)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-star/pull/98.html" title="Last updated on Feb 19, 2021, 5:11 PM UTC (0a9148d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-star/98/f23b97f...0a9148d.html" title="Last updated on Feb 19, 2021, 5:11 PM UTC (0a9148d)">Diff</a>

### CODE REVIEW

**Comment by afs** [comment]
> @afs can you please take a look at this PR.
Yes - will do.

File: cg-spec/editors_draft.html [hunk 1]
```diff
@@ -941,7 +941,206 @@ <h2>SPARQL-star Query Results XML Format</h2>
...
+        <li>all these IRIs are distinct, i.e., |i| ≠ |j| implies that |iri|<sub>|i|</sub> ≠ |iri|<sub>|j|</sub>.</li>
+      </ul>
+
+    </section>
+-->
+
+    <section>
+      <h2>Grammar</h2>
+
+      <p>SPARQL-star Update is an extension of the SPARQL Update language [[SPARQL11-UPDATE]]. As mentioned in <a data-cite="SPARQL11-UPDATE#grammar">SPARQL&nbsp;1.1 Update, Appendix&nbsp;C</a>&nbsp;[[SPARQL11-UPDATE]], the grammar of the latter is provided as part of the <a data-cite="SPARQL11-QUERY#sparrqlGrammar">SPARQL&nbsp;1.1 Query grammar</a>. Similarly, the grammar of SPARQL-star Update is provided as part of the grammar of the SPARQL-star query language, which is defined by the SPARQL&nbsp;1.1 Query grammar with the extensions specified in <a href="#sparql-star-grammar"></a>. As a result of these extensions, the production rules <a data-cite="SPARQL11-QUERY#rQuadData">QuadData</a> and <a data-cite="SPARQL11-QUERY#rQuadPattern">QuadPattern</a>, which are used in the definition of SPARQL Update [[SPARQL11-UPDATE]], are extended to capture nested triples and nested triple patterns as demonstrated in the examples above.</p>
```
Comment (kinow) [inline cg-spec/editors_draft.html:1134]: s/sparrqlGrammar/sparqlGrammar ?

Comment (hartig) [reply cg-spec/editors_draft.html]: Nice catch. Thanks!
Commit https://github.com/w3c/rdf-star/pull/98/commits/817f3746d3167bc00becb6a7ba60b5e076313484 fixes it.

**Comment by gkellogg** [comment]
I think the PR Preview configuration may need to be updated (in .pr-preview.json).

```json
{
    "src_file": "cg-spec/editors_draft.html",
    "type": "respec"
}
```

**Comment by gkellogg** [comment]
> I think the PR Preview configuration may need to be updated (in .pr-preview.json).
> 
> ```json
> {
>     "src_file": "cg-spec/editors_draft.html",
>     "type": "respec"
> }
> ```

See PR #100, which also fixes the CI action.

**Review by pchampin — APPROVED** [review]

File: cg-spec/editors_draft.html [hunk 2]
```diff
@@ -941,7 +941,206 @@ <h2>SPARQL-star Query Results XML Format</h2>
...
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX : <http://www.example.org/>
+
+          DELETE { :alice  ?pp <<?s ?p ?o>> . }
+          INSERT { :alicia ?pp <<?s ?p ?o>> . }
```
Comment (afs) [inline cg-spec/editors_draft.html:1069]: Too subtle! Suggest something more clearly different, like ":carol"

File: cg-spec/editors_draft.html [hunk 3]
```diff
@@ -941,7 +941,206 @@ <h2>SPARQL-star Query Results XML Format</h2>
...
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX : <http://www.example.org/>
+
+          DELETE { :alice  ?pp <<?s ?p ?o>> . }
+          INSERT { :alicia ?pp <<?s ?p ?o>> .  ?s ?p ?o .}
+          WHERE {  :alice  ?pp <<?s ?p ?o>> . }
+          -->
+        </pre>
```
Comment (afs) [inline cg-spec/editors_draft.html:1089]: (for later?) Include the annotation version of this example.

**Review by afs — APPROVED** [review]
I like the exposition.

I have two comments but not ones that need block merging this.

File: cg-spec/editors_draft.html (hunk 2 above)
Comment (hartig) [reply cg-spec/editors_draft.html]: Good point! ;-)
I have changed it to ':carol' in commit https://github.com/w3c/rdf-star/pull/98/commits/b9aae14f4469298c24bd1d832116e8b8b6651fa6

File: cg-spec/editors_draft.html (hunk 3 above)
Comment (hartig) [reply cg-spec/editors_draft.html:1089]: Yes, let's keep this for later (once the issue about the annotation syntax for the query language is settled)

**Comment by hartig** [comment]
As agreed in today's telco, I will merge this PR now.

**Pull request merged**